manifests-maistra/1.1.0/servicemeshmemberrolls.crd.yaml

```diff
@@ -16,7 +16,22 @@ spec:
     status: {}
   version: v1
   additionalPrinterColumns:
-  - JSONPath: .spec.members
+  - name: Ready
+    description: How many of the total number of member namespaces are
+      configured
+    type: string
+    JSONPath: '.status.annotations.configuredMemberCount'
```

Comment:
Would configured members be better?  I think this would be useful for debugging things like why injection hasn't occurred (e.g. project's not configured).

Reply:
MAISTRA-719

Reply:
No. Lists that could potentially be very long don't belong in the default output. I've moved the list of configured members to the `-o wide` output option. But even that's an anti-pattern. The list should be capped at around five elements. If users want to see the full list, they should use `oc describe`.

Reply:
Will those be available in `oc describe`?